### DISCUSSION TITLE
GTAIV.EFLC.Various.Fixes v1.50

### DISCUSSION THREAD

**Discussion by valentyn-l** [valentyn-l/GTAIV.EFLC.Various.Fixes#1] [Announcements]
First repository release, mod version 1.50. 

Please consult the mod's page on [GTA Forums](https://gtaforums.com/topic/975211-various-fixes/) for more updates!

<hr /><em>This discussion was created from the release <a href='https://github.com/valentyn-l/GTAIV.EFLC.Various.Fixes/releases/tag/'>GTAIV.EFLC.Various.Fixes v1.50</a>.</em>

**Comment by forzayt** [comment]
wow
